src/modules/access/access.service.ts

```
@@ -128,250 +60,141 @@ type Role = 'admin' | 'underboss' | 'caporegime' | 'host' | null;
-      .knex('city_access')
-      .where('user_telegram_id', telegram_id)
-      .pluck('city_id');
+    const cityAccess = await this.getCityAccess(telegram_id)
 
-    if (accessibleCityIds.length > 0) {
-      const cities: CityData[] = await this.knexService
-        .knex('city')
-        .whereIn('id', accessibleCityIds)
-        .select('id as city_id', 'name as city_name', 'group_id', 'telegram_link');
+    if (cityAccess.length) {
+      const cities = await this.cityService.getCitiesByCityIds(cityAccess.map((access) => access.id))
```

ESLint

[error] Replace `cityAccess.map((access)·=>·access.id))` with `⏎········cityAccess.map((access)·=>·access.id),⏎······);`

(prettier/prettier)